### ISSUE TITLE
Beautify display of commit messages with bodies and/or footers

### ISSUE THREAD

**Issue by iron3oxide** [woodpecker-ci/woodpecker#1489]
### Clear and concise description of the problem

When committing with a structured message (e.g. following Conventional Commits specification), Woodpecker cramps the whole message together into one large string when displaying it in the repo commit list and the pipeline run title. This is usually pretty ugly and decreases readability. 

Example:
![image](https://user-images.githubusercontent.com/98779754/207985749-39508465-642d-429a-9007-199edf4fad2f.png)

### Suggested solution

A new environment variable CI_COMMIT_MESSAGE_HEADER could be added which would be used to populate the strings mentioned above. This would of course require parsing the commit message first.

### Alternative

Alternatively, a UI section for the body and footer of the commit message could be added below the pipeline run title to display those variables. 

All of the steps described above would still be necessary, but in addition, at least CI_COMMIT_MESSAGE_BODY (if one wants to treat body and footer as one) would have to be parsed and added as well.

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

**Comment by 6543** [comment]
is it the same as https://github.com/woodpecker-ci/woodpecker/issues/1314#issuecomment-1346546424 ?

**Comment by gapodo** [comment]
I'd go with yes and also the same as https://github.com/woodpecker-ci/woodpecker/issues/1314#issuecomment-1302791703 (which is also posted in the ui feedback)

**Comment by iron3oxide** [comment]
Not quite, but it's the same topic for sure. While one of my concerns is the overflow mentioned in the linked comments, that could be caused by a very long commit message header as well.  

The mangling of commit message header, body and footer though is what mainly causes this overflow and also the ugly cap changes in the middle of the text. I'd like to avoid both by using one of the solutions mentioned above, not by cutting the text off at some specified length (which would still leave the cap changes unless one uses periods which looks weird in Github, Gitea etc.)
